### DISCUSSION TITLE
Shallow copy of a retired collection

### DISCUSSION THREAD

**Discussion by kameshbhr** [Sunbird-Knowlg/Community#149] [Q&A] [answered]
When a collection is retired, will it impact it's shallow copy in any way? (I assume the deep copy will not be impacted)

**Comment by ashokreddy1208** [comment, marked as answer]
@kameshbhr we cannot retire a collection if it is adopted by other tenants. we are getting error message as "Content With Identifier [do_ABC] Can Not Be Retired. It Has Been Adopted By Other Users." when we try to delete actual collection.

cc @Krishnaj20
